doc/hintsntips_luna.rst

```diff
@@ -37,16 +37,16 @@ The easiest way to get a new image for modifications is to clone an existing one
 Creating a new image
 ~~~~~~~~~~~~~~~~~~~~
 
-In some other cases you may prefer to start from a fresh image. In that case copy the existing playbook and change name of the image (``image_name`` variable and ``hosts`` target)::
+In some cases it may be preferable to start from a fresh image. In that case, copy the existing playbook and change the name of the image (``image_name`` variable and ``hosts`` target)::
 
     # cp compute.yml new-compute.yml
     # sed -i -e 's/compute/new-compute/' new-compute.yml
 
-You might also want to customize your new image to add more roles. When done, run ``ansible-playbook``::
+In the event an image has been customized/more roles added, run ``ansible-playbook`` once finished::
```

Comment:
Suggestion:
'This new yml file can be customized by adding more roles.

- Now run ansible-playbook::'
(new line for a new important step in the procedure)